src/configure.ac

```diff
@@ -527,6 +531,14 @@ then
 	NVFLAGS="${NVFLAGS} --ptxas-options=-v"
 fi
 
+
+if test "x${enable_cpp11}" != "xno"
```

Comment:
Don't make it by default. For now it will make many problems on several architectures.